packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -389,18 +423,22 @@ export class AuroraPostgresEngineVersion {
    * @param auroraPostgresMajorVersion the major version of the engine,
    *   for example "9.6"
    */
-  public static of(auroraPostgresFullVersion: string, auroraPostgresMajorVersion: string): AuroraPostgresEngineVersion {
-    return new AuroraPostgresEngineVersion(auroraPostgresFullVersion, auroraPostgresMajorVersion);
+  public static of(auroraPostgresFullVersion: string, auroraPostgresMajorVersion: string,
+    auroraPostgresFeatures?: EngineFeatures): AuroraPostgresEngineVersion {
+    return new AuroraPostgresEngineVersion(auroraPostgresFullVersion, auroraPostgresMajorVersion, auroraPostgresFeatures);
   }
 
   /** The full version string, for example, "9.6.25.1". */
   public readonly auroraPostgresFullVersion: string;
   /** The major version of the engine, for example, "9.6". */
   public readonly auroraPostgresMajorVersion: string;
+  /** The supported features for the DB engine */
+  public readonly auroraPostgresFeatures?: EngineFeatures;
```

Comment:
I would probably call it `_features`, and make it `@internal` (I don't see a good reason to increase the surface area here).